pkg/storageos/rbac.go

```diff
@@ -240,6 +240,7 @@ func (s *Deployment) createClusterRoleForScheduler() error {
 				"replicationcontrollers",
 				"pods",
 				"pods/binding",
+				"pods/status",
```

Comment:
The operator also needs this permission before granting it to other resources.
Permission in manifest deployment in [role.yaml](https://github.com/storageos/cluster-operator/blob/1.5.3/deploy/role.yaml#L70) and the OLM CSV deployment [configmap](https://github.com/storageos/cluster-operator/blob/1.5.3/deploy/storageos-operators.configmap.yaml#L874) (optionally also update all the OLM CSV files with `make metadata-update`).